ui/react-components/components/AppointmentEditorCommonFieldsWrapper/AppointmentEditorCommonFieldsWrapper.jsx

```diff
@@ -25,6 +25,16 @@ const AppointmentEditorCommonFieldsWrapper = props => {
     const componentsDisableStatus = props.componentsDisableStatus || {};
     const errorTranslations = getErrorTranslations(intl);
 
+    const updateLocationBasedOnService = (selectedService) => {
+        isEmpty(selectedService.value.location) ? updateAppointmentDetails({location: null})
```

Comment:
Possibility of `undefined`?

Reply:
I see this is already been handled in the below line
https://github.com/Bahmni/openmrs-module-appointments-frontend/blob/feedback-fix/ui/react-components/components/AppointmentEditorCommonFieldsWrapper/AppointmentEditorCommonFieldsWrapper.jsx#L66